static/js/theme-map/Maps/Providers/Mapbox.js

```diff
@@ -51,6 +51,9 @@ class MapboxMap extends ProviderMap {
         this._canvasClickHandler();
       }
     });
+    this.map.on('load', () => {
```

Comment:
is Google and Mapbox the only two map providers we want to fix?

Reply:
Yes we do not support other providers.

Reply:
The other providers are facades and should have been ripped out but the creator of the full page map never had the time

Reply:
> in the techops, there seemed to be a problem with a couple different types of phone. were they all android?

Yes